### PR TITLE
Tp sl update

### PR DESCRIPTION
**Pull request by EricZ11**
take profit stop loss version update

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -551,12 +555,18 @@ en:
...
     <p>Customize conditional order ID: You may customize order IDs for active orders. We will link it to the system order ID , and return the unique system order ID to you after the active order is created successfully. You may use this order ID to cancel your active order. The customized order ID is asked to be unique, with a maximum length of 36 characters.</p>
 
   account_aside_placeCond:
     Take profit/Stop loss is not supported in placing conditional orders. One can only use these 2 functions when placing active orders. Moreover, each account can hold up to 10 conditional orders yet to be filled entirely simultaneously.
+  account_aside_placeCond_linear:
+    <p>Each account can hold up to 10 conditional orders yet to be filled entirely simultaneously.</p>
+    <p>Return value ext_code is not empty,Please refer to <a href="#t-errors">Errors</a></p>
+  account_aside_placeActive_linear:
+    <p>Return value ext_code is not empty,Please refer to <a href="#t-errors">Errors</a></p>
```
Comment (dexter-2) [inline locales/en.yml:568]: Can you clarify what this means? I don't understand.

File: locales/en.yml [hunk 2]
```diff
@@ -1342,18 +1364,29 @@ en:
...
   err_130023: Will be triggered Liq after order is completed
   err_130127: Take Profit, Stop Loss and Trailing Stop Loss are not modified
+  err_130149: SL/TP setting failed.
+  err_130150: Please try again later.
+  err_130151: At least one take profit order in the mode,Cannot be switched
+  err_130152: At least one stop loss order in the mode,Cannot be switched
+  err_130153: Switching failed. Please cancel the current SL/TP setting.
+  err_130154: Switching failed. Please cancel the SL/TP setting of active orders.
+  err_130155: Insufficient quantity required to set TP/SL
+  err_130156: Modify the trigger price when the price and quantity of the active order is not allowed.
```
Comment (dexter-2) [inline locales/en.yml:1374]: Please clarify this error? I don't understand it.

File: locales/en.yml [hunk 3]
```diff
@@ -1342,18 +1364,29 @@ en:
...
+  err_130157: Amendment failed. SL/TP price cannot be amended as order is filled partially.
+  err_130158: SL/TP price cannot be amended under position mode.
+  err_130159: There are more than 20 SL/TP settings in the partial SL/TP mode.
```
Comment (dextertd) [inline locales/en.yml]: Please clarify `err_130156`, `err_130158`, `err_130159`

File: locales/en.yml [hunk 4]
```diff
@@ -607,7 +617,7 @@ en:
   account_aside_changeMargin: You cannot change margin when your position is in Cross Margin mode. See the <a href="#t-changeleverage">Change User Leverage</a> endpoint for more.
   ### Set Trading-Stop
   tradingstop: Set Trading-Stop
-  account_para_tradingStop: Set take profit, stop loss, and trailing stop for your open position.
```
Comment (dextertd) [inline locales/en.yml:609]: Need two versions of this for inverse & linear:
- inverse: `account_para_tradingStop`: "Set take profit, stop loss, and trailing stop for your open position."
- linear: `account_para_tradingStop_linear`: The new version added in this PR.

**So please add `account_para_tradingStop_linear` to zh-cn.yml**

**Comment by dextertd** [comment]
I have a local review ready to push, just waiting on responses to my above comment so that I may make minor adjustments.

**Review by EricZ11 — COMMENTED** [review]
err_130153: Switching failed. There is profit stop loss setting
err_130154: Switching failed. active orders have TP/SL setting.
err_130155: Insufficient quantity
err_130156: replace active order price at the same time replace qty is not allowed
err_130157: active order partial deal，replacec TP/SL is not allowed
  err_130158: TP/SL mode in ‘Full’  , replace tp or sl qty is not allowed
  err_130159: TP/SL mode in ‘Partial’ ,  sum all order take profit  and stop loss number  max is 20

**Review by dextertd — CHANGES_REQUESTED** [review]
Review PR: #107

**Pull request merged**